### ISSUE TITLE
Warnings showing in log files

### ISSUE THREAD

**Issue by richlux** [hluk/CopyQ#1259]
I am seeing the following warning message in the CopyQ log files every time I copy something:

**Warning [2019-11-18 10:10:58.219] <Client-115856>:** Ignoring configuration in "C:/Program Files (x86)/CopyQ/config" (https://github.com/hluk/CopyQ/issues/583).

I originally ran CopyQ portable before I installed it, but I am currently running the installed version.  Also, the C:/Program Files (x86)/CopyQ/config folder doesn't exist.

Thanks,
Rich

**Comment by hluk** [comment]
That's strange, the log gets only printed if the "config" subfolder exists.

Can you try launching using Action dialog from the app (F5 shortcut) following code and see which popup message shows up?

```js
copyq:
popup(Dir("C:/Program Files (x86)/CopyQ/config").exists() ? "EXISTS" : "DOES NOT EXIST")
```

**Comment by richlux** [comment]
It actually says "EXISTS".  If I change the command to look for "configxxx", it returns "DOES NOT EXIST", so the function is working.  Here's a screenshot of that folder.  I am showing Hidden and System files and you can see there is not a "config" folder:

![image](https://user-images.githubusercontent.com/25157703/69069022-c7ae1900-09f3-11ea-8b9a-d76b412b09b3.png)

Rich

**Comment by hluk** [comment]
The folder could be hidden. Try appending `\config` manually to the path in explorer.

**Comment by richlux** [comment]
I am already showing hidden and system files, but I appended the name anyway and it doesn't exists.

![image](https://user-images.githubusercontent.com/25157703/69070727-ab5fab80-09f6-11ea-994b-a51c44004807.png)

Rich

**Comment by hluk** [comment]
Hmm, can you try selecting **Import** from **File** menu in CopyQ and go to the folder? Maybe there are some odd permissions set so that only CopyQ sees the folder.

**Comment by richlux** [comment]
BINGO!  When I looked that way, the folder was showing and I was able to delete it.  No more log entries now!  This is so strange.  I've never seen a folder that didn't show when Windows was set to show hidden and system files.  Thanks for the help.

By the way, I have another issue that I can open a separate ticket for, but I thought I'd ask a quick question here.  I'm trying to debug the "Tab for URLs with Title and Icon" command because it never seems to return any html and therefore can't get the Title and Icon.  I'm debugging with popup() and serverLog(), but if try to use a variable, It always returns the text "NetworkReply(name = "")".  Can I not print out the contents of variables this way. Here's a sample of what I am trying to do:

```
function fetchContent(url, maxRedirects) {
   if (maxRedirects === undefined)
       maxRedirects = 4
   var reply = networkGet(url)
   if (maxRedirects == 0)
       return reply
   var header = findHeader(reply, 'location')
popup(header)
serverLog(header)
   if (header)
       return fetchContent(header, maxRedirects - 1)
   return reply
}
```

Rich
